src/test/cucumber/home.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Print home path
+
+  Background:
+    Given the internet is reachable
+    And an initialised environment without debug prints
```

Comment:
Why without debug prints?

Reply:
Because then I can just do .trim() on the output and compare it to the path I expect. Otherwise I would need to sanitize the output by first removing all the debug prints that occur. My thought was that this was a more robust way of proving that the output I expect is correct. If another error print is added for some reason (not debug) then the test will catch that right away.